scripts/mk-odbc-connector

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+mkdir /usr/src/mariadb-connector-odbc/build
+cd /usr/src/mariadb-connector-odbc/build
+if [ -f /etc/rpi-issue ]; then
+    DM_DIR=/usr/lib/arm-linux-gnueabihf cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCONC_WITH_UNIT_TESTS=Off -DCMAKE_C_FLAGS_RELWITHDEBINFO="-I/usr/include/mariadb -L/usr/lib"
+else
+    DM_DIR=/usr/lib/aarch64-linux-gnu cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCONC_WITH_UNIT_TESTS=Off -DCMAKE_C_FLAGS_RELWITHDEBINFO="-I/usr/include/mariadb" -DWITH_SSL=OPENSSL -DCMAKE_INSTALL_PREFIX=/usr/local
```

Comment:
Is a 3rd case needed for amd64 = x86_64 ?

Reply:
Sure when I know the correct path to use.

Reply:
No rush.  But I can offer VM's to help with testing when we get there.

(With Debian 11 being finalized today, that might be a good one to test.)

Reply:
The latest commit should do it, give it a go

Reply:
The there was never any mention of Freebox being supported on Debian ever, just Ubuntu in the original PR so I might not care if it doesn't work, just another variable I don't want to waste yours and my time on, stick with Ubuntu.

Reply:
Isn't FreePBX better supported on Debian as compared to Ubuntu?

(Certainly that's the impression I get from https://wiki.freepbx.org, where Debian is listed but Ubuntu is not.)

Reply:
My test platform is the RPi-400 running Ubuntu Mate arm64 20.10. which makes Ubuntu a better quick tester platform, should the install work there then I suspect Debian should succeed also but I don't want to debug to two variables at the same time.